storage/engine/bench_test.go

```diff
@@ -363,6 +363,51 @@ func runMVCCBatchPut(emk engineMaker, valueSize, batchSize int, b *testing.B) {
 	b.StopTimer()
 }
 
+func runMVCCBatchTimeSeries(emk engineMaker, batchSize int, b *testing.B) {
```

Comment:
You might want to denote that this test only writes time series. It's a big distinction and we might want to be testing the efficiency of reading at some point, which activates the merge operator.